### ISSUE TITLE
Cannot set/use Broadcast Shortcuts

### ISSUE THREAD

**Issue by BBQigniter** [gnome-terminator/terminator#253]
Just updated to latest version on my Fedora 32 machine. Unfortunately my "Broadcast..." shortcuts are all disabled and if I want to set a shortcut, it stays disabled.

Started terminator in debug-mode - it's showing following on the console

```
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/terminatorlib/prefseditor.py", line 1714, in on_cellrenderer_accel_edited
    parsed_conf_accel = Gtk.accelerator_parse(conf_accel)
TypeError: Argument 0 does not allow None as a value
```

**Comment by BBQigniter** [comment]
as a workaround I edited the keybindings section in the `./config/terminator/config` file 

```
[keybindings]
  zoom_in = None
  zoom_out = None
  broadcast_all = <ALT>A
  broadcast_off = <ALT>O
```

**Comment by mattrose** [comment]
Fixed in #224
